akka-docs-dev/rst/java/http/routing-dsl/directives/host-directives/extractHost.rst

```diff
@@ -0,0 +1,13 @@
+.. _-extractHost-java-:
```

Comment:
We'll need to figure out where to document request values so it's consistent.. For now looks good :)